Scripts/cmake/o3de_package_utilities.cmake

```diff
@@ -0,0 +1,134 @@
+#     It also maps the IMPORTED_LOCATION configurations.  It does so by
+#     checking the imported configurations that are provided for the target ${NAMESPACE_FROM}::${component} 
+#     compared to the configurations that are currently active in the project being configured by CMake.
+#     Any configurations that are present in both are preserved, while any mismatches get defaulted to RELEASE.
+#        For example, if the target provides "Debug", and "Release" imported locations, and the project
+#        being compiled uses "Debug", "RelWithDebInfo", "Release"
+#        it will map "Debug" -> "Debug"
+#                "Release" -> "Release"
+#                "RelWithDebInfo" -> "Release"
+#     Finally, if NAMESPACE_TO is specified, it will create an alias from
+#     {NAMESPACE_FROM}::${component} to ${NAMESPACE_TO}::${component}.
+#     NAMESPACE_TO is usually "3rdParty" for O3DE.
+
+function(o3de_import_targets)
+    set(singleValues NAMESPACE_FROM NAMESPACE_TO)
+    set(multiValues COMPONENTS)
+    
+    cmake_parse_arguments(_o3de_import_targets
+                     ""
+                     "${singleValues}"
+                     "${multiValues}"
+                    ${ARGN})
+
+    if (NOT _o3de_import_targets_COMPONENTS)
+        message(FATAL_ERROR "You must specify COMPONENTS for o3de_import_targets")
+    endif()
+
+    if (_o3de_import_targets_NAMESPACE_FROM)
+        set(from_namespace_prefix "${_o3de_import_targets_NAMESPACE_FROM}::")
+    endif()
+
+    if (_o3de_import_targets_NAMESPACE_TO)
+        set(to_namespace_prefix "${_o3de_import_targets_NAMESPACE_TO}::")
+    endif()
+
+    if (_o3de_import_targets_UNPARSED_ARGUMENTS)
+        message(FATAL_ERROR "Unexpected Arguments: ${_o3de_import_targets_UNPARSED_ARGUMENTS}")
+    endif()
+
+    foreach(component IN LISTS _o3de_import_targets_COMPONENTS)
+        set(component_target_name ${from_namespace_prefix}${component})
+        if(TARGET ${component_target_name})
+
+            # convert the includes to system includes to account for older CMake versions:
+            get_target_property(system_includes ${component_target_name} INTERFACE_INCLUDE_DIRECTORIES)
+            set_target_properties(${component_target_name} PROPERTIES INTERFACE_INCLUDE_DIRECTORIES "") # Clear it in case someone refers to it
+            if (COMMAND ly_target_include_system_directories)
+                ly_target_include_system_directories(TARGET ${component_target_name} INTERFACE ${system_includes})
+            else()
+                target_include_directories(${component_target_name} SYSTEM INTERFACE ${system_includes})
+            endif()
+
+            # Alias the target in the new namespace:
+            if (to_namespace_prefix AND from_namespace_prefix AND NOT "${to_namespace_prefix}" STREQUAL "${from_namespace_prefix}")
+                add_library(${to_namespace_prefix}${component} ALIAS ${component_target_name})
+            endif()
```

Comment:
What if the `from_namespace_prefix` is empty?
In that case an alias target will not be made.

Such as importing a target such as "3rdPartyWithoutNamespace" -> "3rdParty::3rdPartyWithoutNamespace"

Reply:
I'll update that.  In the case of OpenEXR it won't make a difference, so I'm tempted to just keep the existing packages.